spack.yaml

```diff
@@ -1,79 +1,46 @@
-# This is a Spack Environment file.
-#
-# It describes a set of packages to be installed, along with
-# configuration settings.
 spack:
-  # TODO: This section is used primarily to provide variations to a model based on deployment target. Delete if not required.
-  # definitions:
-    # This is the root spack bundle that contains the model
-    # - ROOT_PACKAGE:
-    #   - MODEL@git.VERSION
+  specs:
+    - access-issm@git.2025.03.0
 
-    # Mutually-exclusive Compiler and Target definitions (assumes DEPLOYMENT_TARGET set)
-    # - when: env['DEPLOYMENT_TARGET'] == 'Gadi'
-    #   compiler_target: ['%intel@19.0.5.281 target=x86_64_v4']
-    # More mutually-exclusive definitions can be made...
-    # Default
-    # - when: "'DEPLOYMENT_TARGET' not in env"
-    #   compiler_targets: ['intel@2021.2.0 target=x86_64']
+  packages:
+    issm:
+      require:
+        - '@git.2025.04.11'
+        - +wrappers
 
-    # This is where the single definitions of ROOT_PACKAGE and compiler/target are
-    # matrixed together to form a single spec: MODEL@git.VERSION %intel@2021.2.0 target=x86_64
-    # - ROOT_SPEC:
-    #   - matrix:
-    #     - [$ROOT_PACKAGE]
-    #     - [$%compiler_target]
+    # Mark Python 3.9.2 as external so Spack reuses the system module
+    python:
+      externals:
+      - spec: python@3.9.2
+        modules:
+          - python3/3.9.2
+        prefix: /apps/python3/3.9.2
+      buildable: false
 
-  specs:
-    # The root Spack Bundle Recipe (SBR) for the model and overall version of
-    # the deployment:
-    # TODO: Replace the MODEL and VERSION, if there is only one deployment target
-    # - MODEL@git.VERSION
-    # Alternatively, if there are multiple deployment targets that require
-    # different compilers/variants, use the above definitions section and:
-    # - $ROOT_SPEC
-  packages:
-    # Specification of dependency versions and variants. CI/CD requires that
-    # the first element of the require is only a version:
-    # TODO: Specify versions and variants of dependencies as required.
-    # openmpi:
-    #   require:
-    #     - '@4.1.5'
+    openmpi:
+      require:
+        - '@4.1.7'
 
-    # Specifications that apply to all packages
+    # “all” can remain empty if you aren’t forcing any particular compiler / MPI
     all:
-      # TODO: Specify compiler/targets for all packages
-      # require:
-      #   - '%intel@2021.10.0'
-      #   - 'target=x86_64'
+      require:
+        - '%intel@2021.10.0'
+        - 'target=x86_64'
```

Comment:
Recall this will not actually be applied to any of the packages that have a `require` section. Does it get applied to the `specs`?

Reply:
Hi @aidanheerdegen , This does get applied, that's how we've been able to successfully complete model releases.